weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/CustomGridTreeDataGroupingCell.tsx

```diff
@@ -176,19 +182,32 @@ export const CustomGridTreeDataGroupingCell: FC<
           <>
             <Box
               sx={{
-                mr: 1,
+                display: 'flex',
+                alignItems: 'center',
+                gap: '4px',
               }}>
-              <StatusChip value={row.status} iconOnly />
-            </Box>
-            <Box
-              sx={{
-                overflow: 'hidden',
-                textOverflow: 'ellipsis',
-                whiteSpace: 'nowrap',
-                flex: '1 1 auto',
-              }}>
-              {opNiceName(call.spanName)}
+              <Box
+                sx={{
+                  mr: 1,
+                }}>
+                <StatusChip value={row.status} iconOnly />
+              </Box>
+              <Box
+                sx={{
+                  overflow: 'hidden',
+                  textOverflow: 'ellipsis',
+                  whiteSpace: 'nowrap',
+                  flex: '1 1 auto',
+                }}>
+                {opNiceName(call.spanName)}
+              </Box>
             </Box>
+            {call?.rawSpan?.summary && (
```

Comment:
what does the UI look like when this is not there? (I think this is true for an in-progress call?)